chain/chain/src/chain.rs

```diff
@@ -118,6 +118,12 @@ enum ApplyChunksMode {
     NotCaughtUp,
 }
 
+/// Contains information from preprocessing a block
+struct BlockPreprocessInfo {
+    state_dl_info: Option<StateSyncInfo>,
+    incoming_receipts: HashMap<ShardId, Vec<ReceiptProof>>,
```

Comment:
should we also add apply_chunk_work into here? (or do you prefer to keep it as a separate output form preprocess block?)

Reply:
Yes I want to keep it separated. Later we will need to store BlockPreprocessInfo in chain when we make apply_chunks non-blocking.